scripts/apps/packaging/directives/AddPackageDropdown.ts

```diff
@@ -1,7 +1,8 @@
 import {AuthoringWorkspaceService} from 'apps/authoring/authoring/services/AuthoringWorkspaceService';
+import {IPackagesService} from 'types/Services/Packages';
 
 AddPackageDropdown.$inject = ['$rootScope', 'api', 'packages', 'authoringWorkspace'];
-export function AddPackageDropdown($rootScope, api, packages, authoringWorkspace: AuthoringWorkspaceService) {
+export function AddPackageDropdown($rootScope, api, packages: IPackagesService, authoringWorkspace: AuthoringWorkspaceService) {
```

Comment:
I think this one is too long still

Reply:
I'll fix it. How can the tests pass on my local copy and not here? 🤔

Reply:
I think test instances run tests, not against your branch, but against target branch(develop in this case) + your branch merged. Because of this they can pass locally, but still fail on the test server. I'm not sure if it works this way with unit tests, but it does with e2e tests.